server/dba/seeders/prod/02-species.js

```diff
@@ -114,12 +114,54 @@ module.exports = {
         web_url: null,
         created: 'now()',
         updated: 'now()'
+      },
+      {
+        id: 17,
```

Comment:
This still feels like considerable overarchitecting to have a 1:many relationship for species/species notes. When you could just have a paragraph.

Reply:
@lauraGgit created a new story for this: https://github.com/nciinc/fs-intake-module/issues/104